src/lib/components/cloudprogressbar/CloudProgressBar.component.js

```diff
@@ -0,0 +1,132 @@
+//@flow
+import React from 'react';
+import styled, { css, keyframes } from "styled-components";
+import * as defaultTheme from "../../style/theme";
+import type { Node } from "react";
+
+export type CloudProgressBarProps = {
+  percentage: number,
+  borderSize: string,
+  cloudColor?: string,
+  progressCloudColor?: string,
+  children: Node
+};
+
+const keyFrameCloud = props => {
+  return keyframes`
+    from {
+      height:100%
+    } 
+    to {
+      height:${100 - props.percentage}%
+    }`;
+};
+
+const Container = styled.div`
+  ${props => {
+    return css`
+      width: 100%;
+      text-align: center;
+      position: relative;
+    `;
+  }}
+`;
+
+const ContainerProgress = styled.div`
+  ${props => {
+    return css`
+      animation-duration: 1s;
+      animation-fill-mode: both;
+      animation-name: ${keyFrameCloud};
+      position: absolute;
+      top: 0;
+      left: 0;
+      display: block;
+      width: 100%;
+      overflow: hidden;
+    `;
+  }}
+`;
+
+const CloudContainer = styled.div`
+  position: relative;
+`;
```

Comment:
I think we could remove the `<CloudContainer/>` and the style as well

Reply:
Nop. We need of this to align the items, and for the two clouds be aligned as well.

Reply:
Position absolute means
> The element is positioned relative to its first positioned (not static) ancestor element.

If we remove the `<cloudContainer/>`, it's ancestor is `<container/>`which we already set ` position: relative;` So I think if we remove `<cloudContainer/>` should also work!

I will let you try it ;)